backend/spec/features/admin/taxons_spec.rb

```diff
@@ -76,6 +76,14 @@ def only_one_selected_tab_inside?(sub_tab_selector, tab_name, tab_path)
     end
   end
 
+  it "should not send exists to attachments" do
+    taxon = create(:taxon)
+
+    expect_any_instance_of(Paperclip::Storage::Filesystem).not_to receive(:exists?)
+
+    visit spree.edit_admin_taxonomy_taxon_path(taxon.taxonomy, taxon.id)
+  end
```

Comment:
This test is _extremely_ fragile and totally dependent on the implementation details of a deprecated gem. I like the change itself, but as I see it this test guards against nothing other than someone reverting this exact change which I don't think we need to worry about.

I'm open to hearing alternative perspectives on it, but I don't currently see any value in maintaining this test.

Reply:
I agree, the test here is not very meaningful. The only alternative I see is testing this in integration by creating a record that has an attachment, but whose the attachment doesn't actually exist. However, I think it would be much more effort than the value we'd get out of it.

Reply:
Makes sense to me! I've ditched the spec, let me know if that works for you.